pkg/fanal/cache/s3.go

```diff
@@ -17,14 +17,20 @@ import (
 
 var _ Cache = &S3Cache{}
 
+type s3API interface {
```

Comment:
v2 no longer provides interfaces like v1?

Reply:
No, it doesn't. They just have interfaces for subparts of the API but it's incomplete (missing HeadObject and DeleteBucket) so I had to create it.